charger/fritzdect.go

```diff
@@ -34,6 +34,14 @@ func NewFritzDECTFromConfig(other map[string]interface{}) (api.Charger, error) {
 		return nil, err
 	}
 
+	if cc.User == "" || cc.User == "<nil>" {
```

Comment:
Das soll/darf so nicht sein. Woher kommt das "<nil>"?

Reply:
util.DecodeOther liefert in einem Paramterwert "<nil>" zurück, wenn dieser ohne Wert konfiguriert wird.

Beispiel:
```yaml
...
  user:
  password: xxxxxxxxx
...
```
Die Konfiguration oben würde nach `util.DecodeOther` folgende Werte liefern:
```
cc.URI = "" (weil die Parameter spec komplett fehlt)
cc.User = "<nil>"
cc.Password = "xxxxxxxxx"
```

Reply:
Ich schaue mir die Decoder Funktion mal an.